### ISSUE TITLE
No Bounding tile present

### ISSUE THREAD

**Issue by kirenesnah** [EvanesceExotica/Journal-To-Canvas-Slideshow#15]
Hi EvanesceExotica,
I'm having an issue where my display images are not staying in my bounding tile. I put the bounding tile down and for some reason it doesn't recognize that there is one. As my screenshot shows I do have a bounding window in the scene, but it isn't staying inside. Then I tried the bounding tile in a different scene like your youtube video showed and it gives me a warning up top. Let me know what you think I can do to fix this. 
Thanks!
![FoundryVTT_Vb7nSOfJGp](https://user-images.githubusercontent.com/45153491/115307488-e5266b80-a12e-11eb-80fe-f932a6ecf853.jpg)
![FoundryVTT_Lbcxo3Sfw9](https://user-images.githubusercontent.com/45153491/115307501-e8b9f280-a12e-11eb-810e-441bd7805b3d.jpg)
![FoundryVTT_i4mCAqRU5n](https://user-images.githubusercontent.com/45153491/115307505-e9eb1f80-a12e-11eb-9f83-341e3f4df6b9.jpg)

**Comment by benbarbour** [comment]
I'm having the same issue. Bounding tile seems to be required but ignored.

**Comment by EvanesceExotica** [comment]
Sorry for the late reply! My last few weeks were filled with final exams in school so I've been pretty busy :( 

I think the issue you might be running into is that right now, the scene you want the bounding tile in needs to be active when you click on the journal image. I realized I wrote my code with that restriction, but I didn't specify that in the warning message. Apologies! I'm pretty new to this and have a lot to learn haha. 

I'd be glad to change that in my next update simply looks for the scene that the tile is within rather than the scene that is active, as I realize that's not ideal.  

For now, try activating the scene with the bounding tile when trying to change the image, and see if that works.

I could be wrong and it could be something else that's causing the problem, but looking at your screenshots, a different scene appears to be active, so trying to activate the scene with the bounding tile will be the first thing to try.
